index.md

```diff
@@ -1,8 +1,3 @@
-> Note that the API is still **alpha**.
-> It is not yet stable and may change at any time without prior notice.
-
-*A [Prototype Fund](https://prototypefund.de/project/emissions-api/) project.*
```

Comment:
Did you remove this line on purpose?

Reply:
Yes. I've originally added it to link to the Prototypefund.
By now, we have included their logo and this looked a bit out of place.
That's why I dropped this.

I should have put this into a separate pull request though. Sorry.